access/slack/README.md

```diff
@@ -24,6 +24,7 @@ root_cas = "/var/lib/teleport/plugins/slack/auth.cas"    # Teleport cluster CA c
 token = "api-token"         # Slack Bot OAuth token
 secret = "secret-value"     # Slack API Signing Secret
 channel = "channel-name"    # Message delivery channel
+readonly = false            # Whether run the plugin in notification mode (Don't allow approval/denial via a Slack button)
```

Comment:
I would change this to `notification_only = true` to be more explicit of what's being turned off, we should keep it as false as default and not require it to be defined.

Reply:
Done!